corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html

```diff
@@ -12,9 +12,9 @@ <h4 class="alert-heading">
                     <span>
                         {% case error.type "blank form" %}
                             {% if not error.message %}
-                                <i class="fa fa-plus"></i> <strong>Add Question</strong> to the
+                                <strong>Add a Question</strong> to the
```

Comment:
by non trivial...I mean to do with properly we'd have to `{%with foo as bar %}` a lot of the variables. and restructure the if statements, otherwise we'll get a lot of loosely contexted translations